### PR TITLE
Added the section for routing transitions

### PR DESCRIPTION
**Pull request by AchrafBardan**
This is my first contribution but this is very important because it is not written any where in the documentation.

### CODE REVIEW

File: content/docs/en/routing/manual-routing.md [hunk 1]
```diff
@@ -115,6 +115,39 @@ this.$navigateTo(Detail, {
 
 For more information about the options that you can pass, see [`NavigationEntry`](https://docs.nativescript.org/api-reference/interfaces/_ui_frame_.navigationentry).
 
+#### Using transitions while navigating
```
Comment (rigor789) [inline content/docs/en/routing/manual-routing.md:118]:
```suggestion
#### Specifying a transition
```

File: content/docs/en/routing/manual-routing.md [hunk 2]
```diff
@@ -115,6 +115,39 @@ this.$navigateTo(Detail, {
...
+
+You can use multiple transitions when routing.
+Can be one of the built-in transitions:
```
Comment (rigor789) [inline content/docs/en/routing/manual-routing.md:121]:
```suggestion
You can use any of the built-in transitions:
```

File: content/docs/en/routing/manual-routing.md [hunk 3]
```diff
@@ -115,6 +115,39 @@ this.$navigateTo(Detail, {
...
+For example:
+
+```JavaScript
+this.$navigateTo(Detail, {
+  transition: {
+    name: "slideLeft",
+    duration: 300,
+    curve: "easeIn"
+  },
+
```
Comment (rigor789) [inline content/docs/en/routing/manual-routing.md:146]:
```suggestion
```

**Review by rigor789 — APPROVED** [review]
Awesome! Thanks!

**Comment by AchrafBardan** [comment]
Thank you for using my contribution on this amazing framework!

**Pull request merged**